aiozmq/interface.py

```diff
@@ -194,3 +223,12 @@ def msg_received(self, data):
 
         data is the multipart tuple of bytes with at least one item.
         """
+
+    def event_received(self, event):
+        """Called when a ZeroMQ socket event is received.
+
+        This method is only called when a socket monitor is enabled.
+
+        :param event: A dict containing the event description keys `event`,
```

Comment:
I dislike `dict` (yes, it's supported by `zmq.utils.monitor.parse_monitor_message()` but we may repeat these 6 lines of code.
My suggestion is using `namedtuple` instead.